src/number_theory/bernoulli.lean

```diff
@@ -226,6 +226,13 @@ end
 /-- The Bernoulli numbers are defined to be `bernoulli'` with a parity sign. -/
 def bernoulli (n : ℕ) : ℚ := (-1)^n * (bernoulli' n)
 
+@[simp] lemma bernoulli'_of_bernoulli (n : ℕ): bernoulli' n  = (-1)^n * bernoulli n :=
```

Comment:
We usually use `of` to indicate that something is a hypothesis. The following name is a bit long, but it's the best I could think of. Maybe there are other suggestions?
```suggestion
lemma bernoulli'_eq_neg_one_pow_mul_bernoulli (n : ℕ) : bernoulli' n = (-1)^n * bernoulli n :=
```